ctcdecode/src/ctc_beam_search_decoder.cpp

```diff
@@ -35,7 +36,9 @@ std::vector<std::pair<double, Output>> ctc_beam_search_decoder(
   // size_t blank_id = vocabulary.size();
 
   // assign space id
-  auto it = std::find(vocabulary.begin(), vocabulary.end(), " ");
+  // Changed by Gideon from the blank symbol " " to a custom symbol specified as argument
```

Comment:
Dead code tells no lies. Please remove this comment and the line of code you commented out below (line 41). That's what git history is for.